src/content/docs/AnnotatedResearchContext.md

```diff
@@ -105,18 +105,18 @@ Name | Type | Tasks on metadata
 
 <!-- Knowledge base cross-references -->
 
-[kb-git]: ./VersionControlGit.html "Git"
-[kb-datahub]: ./DataHub.html "DataPLANT DataHUB"
-[kb-FAIR]: ./FairDataPrinciples.html "FAIR Data principles"
-[quickstart-arc]: ./QuickStart_arc.html "Quickstart ARC"
-[kb-Repositories]: ./PublicDataRepositories.html "Repositories"
-[kb-RDM]: ./ResearchDataManagement.html "Research Data Management"
+[kb-VersionControlGit]: ./VersionControlGit.html "Git"

```

Comment:
I don't really see why you changed all of the abbreviations in this file. Because you want to use short forms for them i think in this case it is fine to split markdown file name/url from abbreviation. But if you want the change we can do it.

Reply:
I just wanted to unify as much as possible article title - file name - sidebar.